[BW] Profile page

GenisDiaz opened the pull request:
**Issue(s)**
[Profile](https://github.com/MTG/freesound/issues/1498)

**Description**

1. Create profile page
- Taps Functionality between Sounds, packs and downloaded
2. Create followers, following and tag following pages

ffont commented:
Thanks @GenisDiaz for the PR!! Here are my comments:

Profile
- [ ] navbar should have the "search sounds" widget at the top left (next to the logo) - this is already implemented in the front page ([example here](https://projects.invisionapp.com/d/main#/console/11781900/248591623/preview))
- [ ] The separators in "sounds packs download" row are invisible (but I see them in the invision version) - I guess this was fixed in browse page and will be fixed here as well once merged to this branch?
- [ ] The tags in the sound player are bolder, and also the spacing in the stars and some other things in the player are not right. I think this was fixed in furthter edits in the search page, so maybe this will be fixed when merging
- [ ] The numbers of main user stats shuuld be black and the spacing does not look right for the "average rare" one
![Screenshot 2020-11-25 at 08 02 52](https://user-images.githubusercontent.com/478615/100194951-df698880-2ef6-11eb-9171-056e51fa4b8c.png)
- [ ] Username font size is different, also the rounded buttons and font weight seem to be a bit off 
![Screenshot 2020-11-25 at 08 05 21](https://user-images.githubusercontent.com/478615/100194990-f4461c00-2ef6-11eb-92c7-8c7b16b1f44f.png)

Followers and following
- [ ] The rounded buttons are also a bit different in size and font color/weight. I guess fixing this in one place it will fix it everywhere.
- [ ] The dividers are also almost invisivble - I guess this was fixed in browse page and will be fixed here as well once merged to this branch?
- [ ] Font weight in user "description" is also bodler
- [ ] Missing red "following" button, although I guess this is just a matter of using the right class
![Screenshot 2020-11-25 at 08 06 54](https://user-images.githubusercontent.com/478615/100195222-4dae4b00-2ef7-11eb-9b96-62b32b0f58f4.png)

Tags following
- [ ] Tags have bolder font and the spacing does not see right. Also tags are not centered.
![Screenshot 2020-11-25 at 08 10 11](https://user-images.githubusercontent.com/478615/100195230-50a93b80-2ef7-11eb-9446-ed9d8d717432.png)

GenisDiaz commented:
@ffont @alastair I've fixed all the comments.
- the dividers are fixed in the other branch
- I've added the following red button in the followers/following page
